### PR TITLE
Fills In Previously Skipped Unit Tests for revive.js

### PR DESCRIPTION
**Pull request by rrgayhart**
### Changes

Fills in unit tests that were skipped/pending for `revive.js`.

----
UUID: 4f82317e-6f0c-4855-9512-3b9b8ca0017d

### CODE REVIEW

File: test/common/ops/revive.js [hunk 1]
```diff
@@ -53,7 +53,21 @@ describe('shared.ops.revive', () => {
     expect(user.stats.str).to.equal(1);
   });
 
-  it('TODO: test actual ways stats are affected');
+  it('it decreases a random state from str, con, per, int by one', () => {
```
Comment (crookedneighbor) [inline test/common/ops/revive.js]: state -> stat

File: test/common/ops/revive.js [hunk 2]
```diff
@@ -65,15 +79,55 @@ describe('shared.ops.revive', () => {
...
-  it('does not remove 0 value items');
+  it('does not remove 0 value items', () => {
+    let eyewearKey = 'eyewear_special_yellowTopFrame';
+    user.items.gear.owned[eyewearKey] = true;
 
-  it('allows removing warrior sword (0 value item)');
+    let [, message] = revive(user);
 
-  it('does not remove items of a different class');
+    expect(message).to.equal('');
```
Comment (crookedneighbor) [inline test/common/ops/revive.js]: I don't think it's important to assert on the empty message here.

File: test/common/ops/revive.js [hunk 3]
```diff
@@ -65,15 +79,55 @@ describe('shared.ops.revive', () => {
     expect(user.items.gear.owned[weaponKey]).to.be.false;
   });
 
-  it('does not remove 0 value items');
+  it('does not remove 0 value items', () => {
+    let eyewearKey = 'eyewear_special_yellowTopFrame';
+    user.items.gear.owned[eyewearKey] = true;
```
Comment (crookedneighbor) [inline test/common/ops/revive.js]: To make sure that this is the only owned item, let's do:

``` js
user.items.gear.owned = {
  eyewear_special_yellowTopFrame: true, // eslint-disable-line camelcase
};
```

It's okay to use the full key name when setting it up and making the assertion. Not DRY, but it makes the test easier to read.

File: test/common/ops/revive.js [hunk 4]
```diff
@@ -65,15 +79,55 @@ describe('shared.ops.revive', () => {
...
 
-  it('does not remove items of a different class');
+    expect(message).to.equal('');
+    expect(user.items.gear.owned[eyewearKey]).to.be.true;
+  });
 
-  it('removes "special" items');
+  it('allows removing warrior sword (0 value item)', () => {
+    let weaponKey = 'weapon_warrior_0';
+    user.items.gear.owned[weaponKey] = true;
```
Comment (crookedneighbor) [inline test/common/ops/revive.js]: Same comment as my previous comment.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]
Overall, 👍 . Just a few nitpicky comments.

**Comment by rrgayhart** [comment]
@crookedneighbor 👍 - changes made - lmk if you see more revisions!

**Comment by crookedneighbor** [comment]
Notated toward your next tier.

**Pull request merged**